eclass/cuda.eclass

```diff
@@ -81,7 +81,7 @@ cuda_gccdir() {
 
 	if [[ -n ${gcc_bindir} ]]; then
 		if [[ -n ${flag} ]]; then
-			ret="${flag}\"${gcc_bindir}\""
+			ret="${flag}${gcc_bindir}"
```

Comment:
I just took another look at this @mgorny.
${flag}="--compiler-bindir"
ret="${flag}\"${gcc_bindir\"", thus you get something like this
ret="\"--compiler-bindir\"\"${gcc_bindir\""
This is too many inline quotes and the source of the problem. ${flag} does not need to be encapsulated by "". Only bindir.
Fixed.
Thank you for your feedback, drove to figure this out. Sorry for the late response.
I hope that this suffices. I'm impressed that there aren't more people complaining about this...guess everyone just changes the compiler or something.